third_party/move/move-compiler-v2/tests/ability-transform/borrowed_from_one_path.exp

```diff
@@ -71,6 +71,50 @@ fun m::f($t0: u8, $t1: &vector<u64>): u64 {
  14: return $t2
 }
 
+============ after LiveVarAnalysisProcessor: ================
```

Comment:
Why are we printing debugging info in test outputs?

Reply:
It's the original test configuration for the ability transformer. We have this new debugging info since we run one more live var analysis.